.circleci/config.yml

```diff
@@ -0,0 +1,80 @@
+version: 2.1
+
+executors:
+  node:
+    parameters:
+      tag:
+        type: string
+    docker:
+      - image: cimg/node:<< parameters.tag >>
+
+commands:
+  install-dependencies:
+    steps:
+      - run:
+          name: "Install project dependencies"
+          command: npm i
+  apt-update:
```

Comment:
This is a bit overdoing it - just have it as a step in `install-prerequisites` (as in, run `apt-get update` as a `step` within that command)